src/components/30-organisms/modal/index.js

```diff
@@ -0,0 +1,98 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import closeIcon from '@axa-ch/materials/internal-icons/close.svg';
+
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAModal extends LitElement {
+  static get tagName() {
+    return 'axa-modal';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      open: { type: Boolean },
+      closeButtonText: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  render() {
+    const classes = {
+      'o-modal--open': this.open,
+    };
+    return html`
+      <article class="o-modal ${classMap(classes)}">
+        <div class="o-modal-window__upper-close-container">
```

Comment:
This is wrong. This is not BEM. It should be something like `o-modal__window__upper-close-container`.

This also counts for most other classes.